aliases/available/bash_it.aliases.bash

```diff
@@ -0,0 +1,24 @@
+cite about-alias
```

Comment:
Can you please change the file name to `bash-it.aliases.bash` (with a dash, not an underscore)?